package.json

```diff
@@ -174,17 +174,17 @@
     "react-native-nitro-modules": "0.24.1",
     "react-native-nitro-sqlite": "9.1.9",
     "react-native-onyx": "^2.0.105",
-    "react-native-pager-view": "6.5.2",
+    "react-native-pager-view": "6.5.3",
```

Comment:
Can this be upgraded separately?

Reply:
Yes, App PR: https://github.com/Expensify/App/pull/62576